script/UpdateDeBridgeFacet.s.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { UpdateScriptBase } from "./utils/UpdateScriptBase.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import "forge-std/console.sol";
```

Comment:
not so important but could be removed

Reply:
I think it should be removed in other PR since other scripts have it in them as well.
so we can create a new PR to remove this line from all scripts

Reply:
as you prefer